### PR TITLE
Add whitepaper and primer updating notes

### LINKED ISSUE #17: Capture our key use cases and design decisions in the CDEvents primer document
Capture our key use cases and design decisions in the CDEvents primer document

### PR DESCRIPTION
**Pull request by afrittoli**
Add the white paper as PDF to the repo, including a paragraph about it in the primer.md

Document that decision decisions are included in primer.md, and add a pull request template to remind PR authors about it

Fixes #17

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

File: primer.md [hunk 1]
```diff
@@ -10,16 +10,25 @@ description: >
...
 
-This non-normative document provides an overview of the CDEvents specification. It is meant to complement the CDEvents specification to provide additional background and insight into the history and design decisions made during the development of the specification. This allows the specification itself to focus on the normative technical details.
+This non-normative document provides an overview of the CDEvents specification.
+It is meant to complement the CDEvents specification to provide additional
+background and insight into the history and design decisions made during the
+development of the specification. This allows the specification itself to focus
+on the normative technical details.
+
+When new features are added to the CDEvents specification, the CDEvents primer
+document is updated accordingly to reflect the design decisions behind the change.
```
Comment (afrittoli) [inline primer.md:20]: Added

File: primer.md [hunk 2]
```diff
@@ -91,46 +102,69 @@ attribute and provide the corresponding JSON object through the
...
+[lowerCamelCase](https://en.wikipedia.org/wiki/Camel_case) for readability
+purposes.
+
+## Whitepaper
+
+The [CDEvents whitepaper](./CDEvents_Whitepaper.pdf) has been originally
+[published][whitepaper] on June 7, 2022 on the CDF blog. It is intended for an
+audience of DevOps Engineers, Project Managers/Directors, CTOs, and Cloud
+Architects who are interested in learning more about CDEvents, why it was
+created and its mission.
```
Comment (afrittoli) [inline primer.md:163]: Added

**Review by e-backmark-ericsson — APPROVED** [review]

**Comment by e-backmark-ericsson** [comment]
If we want to update the whitepaper, in what format do we update it? We will probably not update the PDF file itself right? Should we store the source of the whitepaper in this repo as well?

**Comment by afrittoli** [comment]
We don't have the source of the white paper in a text format. Over time we may convert it to mark down, but for now I would like to have it in the repo, and adding the PDF seems like an ok option.

**Comment by e-backmark-ericsson** [comment]
ok

**Pull request merged**